build.sbt

```diff
@@ -58,9 +58,15 @@ lazy val bloopShared = project
     libraryDependencies ++= Seq(
       Dependencies.jsoniterCore,
       Dependencies.jsoniterMacros,
-      Dependencies.bsp4s excludeAll ExclusionRule(
-        organization = "com.github.plokhotnyuk.jsoniter-scala"
+      Dependencies.bsp4s.excludeAll(
+        ExclusionRule(
+          organization = "com.github.plokhotnyuk.jsoniter-scala"
+        ),
+        ExclusionRule(
+          organization = "me.vican.jorge"
+        )
       ),
+      Dependencies.jsonrpc4s,
```

Comment:
So I guess we should also switch the jsonrpc4s version in [build-server-protocol](https://github.com/build-server-protocol/build-server-protocol)?

Reply:
Hmm... that might be an option too, but it's only really used by Bloop currently as far as I know